chia/pools/pool_wallet.py

```diff
@@ -320,7 +324,7 @@ async def rewind(self, block_height: int, in_transaction: bool) -> bool:
 
     @staticmethod
     async def create(
-        wallet_state_manager: Any,
+        wallet_state_manager: "WalletStateManager",
```

Comment:
Note: technically, you only need
    `from __future__ import annotations`
OR
    the type name as a string

However, this plays nicer with some tools that perform type checks on the fly, as you are editing the file.